profile.js

```diff
@@ -95,21 +99,36 @@ events.on('test', function (data) {
 });
 
 events.on('results', function (data) {
-	console.profileEnd();
+	if (console.profileEnd) {
+		console.profileEnd();
+	}
 	console.log('RESULTS:', data.stats);
+
+	if (process.exit) {
+		process.exit(data.stats.failCount + uncaughtExceptionCount); // eslint-disable-line
```

Comment:
This could also be fixed by making this file executable and adding a hashbang line.

Does `iron-node`, etc. tolerate hash bangs?